packages/utils/src/common/sharding.ts

```diff
@@ -0,0 +1,12 @@
+import type { PubSubTopic } from "@waku/interfaces";
+
+export function ensurePubsubTopicIsValid(
```

Comment:
I don't think _valid_ is the right term here as no validation check is done.

```suggestion
export function ensurePubsubTopicIsConfigured(
```
```suggestion
export function ensurePubsubTopicIsIncluded(
```